src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java

```diff
@@ -303,28 +333,24 @@ private long formatProperties(PrintWriter out, PropertyContainer pc, Long id) {
         return Iterables.count(pc.getPropertyKeys());
     }
 
-    private String formatPropertyName(String prop) {
-        return ", `" + prop + "`:";
-    }
-
     private String formatProperties(PropertyContainer pc, Long id) {
```

Comment:
I think this should go into typeExport as well, perhaps the superclass

and probably change the pc.getPropertyKeys into pc.getProperties and use the map internally